crates/astria-telemetry/Cargo.toml

```diff
@@ -13,6 +13,7 @@ homepage = "https://astria.org"
 [dependencies]
 base64 = { workspace = true, optional = true }
 base64-serde = { workspace = true, optional = true }
+const_format = { version = "0.2.32", optional = true }
```

Comment:
I think it'd be better to not make this optional.  Having created the macro to avoid copy-paste errors/typos, I think it should be the default way to declare metric names going forwards.

Reply:
Removed the `macros` feature and made this non-optional in [e64ca43](https://github.com/astriaorg/astria/pull/1129/commits/e64ca43d377e20cad7621e12f11821b979e5dd14)